support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -66,7 +66,7 @@ class SendAcquisitionEvent(serviceProvider: ServiceProvider = ServiceProvider)
       _ => HandlerResult((), requestInfo)
     )
 
-    val cloudwatchEvent = paymentSuccessRequest(Configuration.stage, paymentProviderFromPaymentMethod(state.paymentMethod), state.product)
+    val cloudwatchEvent = paymentSuccessRequest(Configuration.stage, state.paymentMethod.map(paymentProviderFromPaymentMethod), state.product)
```

Comment:
as per earlier, looks like it does make sense for no payment method, but it should be more like subscriptionSuccessRequest.  Ideally the metric should be renamed also, but there would be a bit more work checking the alarms as well.